drivers/storage/portworx/manifest/manifest.go

```diff
@@ -207,16 +207,16 @@ func (m *manifest) GetVersions(
 
 	cacheExpired := m.lastUpdated.Add(refreshInterval(cluster)).Before(time.Now())
 	if _, ok := provider.(*configMap); !ok && !cacheExpired && !force {
-		return m.cachedVersions.DeepCopy()
+		return m.cachedVersions.DeepCopy(), nil
 	}
 
 	// Bug: if it fails due to temporarily network issue, we should retry.
 	rel, err := provider.Get()
 	if err != nil {
-		msg := fmt.Sprintf("Using default version due to: %v", err)
+		msg := fmt.Sprintf("Portworx install stopped as versions from Configmap cannot be fetched and URL unreachable due to: %v", err)
 		logrus.Error(msg)
 		m.recorder.Event(cluster, v1.EventTypeWarning, util.FailedComponentReason, msg)
-		return defaultRelease(m.k8sVersion)
```

Comment:
Let' remove func defaultRelease if we are not using it anymore anywhere in the code.